SSLeay.xs

```diff
@@ -6363,7 +6367,6 @@ RSA_generate_key(bits,ee,perl_cb=&PL_sv_undef,perl_data=&PL_sv_undef)
        rc = RSA_generate_key_ex(ret, bits, e, new_cb);
        BN_GENCB_free(new_cb);
 #else
-       BN_GENCB new_cb;
```

Comment:
https://github.com/radiator-software/p5-net-ssleay/pull/188 has similar changes except of this line. The changes in https://github.com/radiator-software/p5-net-ssleay/pull/188 look useful too, I looked at them just recently, therefore if this is merged it creates a small conflict, but nothing that can't be managed when processing https://github.com/radiator-software/p5-net-ssleay/pull/188.